app/src/main/java/com/example/cookbook/ui/create_recipe/OwnRecipesViewModel.kt

```diff
@@ -3,18 +3,19 @@ package com.example.cookbook.ui.create_recipe
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
-import androidx.lifecycle.viewModelScope
 import com.example.cookbook.domain.models.RecipeData
 import com.example.cookbook.domain.repository.OwnRecipeRepository
 import com.example.cookbook.utils.Constants.Companion.EMPTY_STRING
 import com.example.cookbook.utils.Constants.Companion.ZERO
-import kotlinx.coroutines.launch
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
 import javax.inject.Inject
 
 class OwnRecipesViewModel @Inject constructor(private val recipeRepository: OwnRecipeRepository) :
     ViewModel() {
 
-    var ownRecipes: LiveData<List<RecipeData>>? = null
+    val ownRecipes: MutableLiveData<List<RecipeData>> = MutableLiveData()
```

Comment:
Let's add private livedata to this `ownRecipes` with `get()` method to avoid change value of this livedata from another classes